src/HiringPromo.js

```diff
@@ -1,14 +1,20 @@
 import React from 'react'
 import {Box, Heading, Text} from '@primer/components'
-import Octicon, {RadioTower} from '@githubprimer/octicons-react'
+import styled from 'react-emotion'
+import {textAlign} from 'styled-system'
+import Octicon from './Octicon'
+import {RadioTower} from '@githubprimer/octicons-react'
 import ButtonPromo from './ButtonPromo'
 import LinkPromo from './LinkPromo'
 
+// FIXME: when textAlign is added to LAYOUT props, we can remove this
+const AlignBox = styled(Box)(textAlign)
+
 export default function HiringPromo() {
   return (
-    <Box align="center" px={5} className="container-lg text-left text-md-center">
+    <AlignBox px={5} textAlign={['left', 'left', 'center']} className="container-lg">
```

Comment:
Ah, I didn't realize we didn't have `textAlign` yet.  Nbd since will need follow up, but I'd extend the `Text` component rather than `Box` here as it's a typography prop not a layout prop, and it would achieve the same result.